### ISSUE TITLE
problem about fine tuning on UCF101 dataset

### ISSUE THREAD

**Issue by JosephChenHub** [HolmesShuan/ResNet-18-Caffemodel-on-ImageNet#2]
Can you show your training log about this model? I'm exploiting it  as a pre-training model for fine tuning on UCF101 RGB dataset, the learning rate starts with 0.002,decreases to its 1/10 every 10,000 iterations,stops at 30,000 iterations,max iterations is 150,000.But the result is not pleasant,the loss decreases slowly as well as the error of test.

**Comment by HolmesShuan** [comment]
From what I can tell, fine tuning seems to be quite intuitive. I can't be pretty sure whether this model will be suitable for Human Action Classification task. Hopefully, it will shorten the learning time but not a guarantee of best model.
